apps/E2E/src/Checkbox/pages/CheckboxPageObject.ts

```diff
@@ -5,61 +5,52 @@ import {
   HOMEPAGE_CHECKBOX_BUTTON,
   CHECKBOX_ON_PRESS,
 } from '../consts';
-import { BasePage, By } from '../../common/BasePage';
-
-/* This enum gives the spec file an EASY way to interact with SPECIFIC UI elements on the page.
- * The spec file should import this enum and use it when wanting to interact with different elements on the page. */
-export const enum CheckboxSelector {
-  Primary, //this._primaryComponent
-}
+import { BasePage, By, DesktopPlatform } from '../../common/BasePage';
+import { Attribute, AttributeValue } from '../../common/consts';
 
 class CheckboxPageObject extends BasePage {
   /******************************************************************/
   /**************** UI Element Interaction Methods ******************/
   /******************************************************************/
   async isCheckboxChecked(): Promise<boolean> {
-    return await (await this._primaryComponent).isSelected();
+    const checkbox = await this._primaryComponent;
+    if (this.platform === DesktopPlatform.Windows) {
```

Comment:
Was this failing before on UWP?

Reply:
Yes. Previously, the functional tests were commented out because `isSelected()` did and **still does** return false for looking at Checkbox state.

Reply:
Do we know why? Any investigation?

Reply:
Ran the UWP test. Appium calls the method: `GET /session/:sessionId/element/:id/selected`, and WinAppDriver always returns `false` despite the checkbox state. 

<img width="855" alt="image" src="https://user-images.githubusercontent.com/15683103/210460551-12bfc0c8-534d-47b8-9ab4-014c90666aa8.png">

Can't test with RadioGroup because the page doesn't exist on the UWP app.